release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -286,6 +286,75 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 [id="ocp-4-10-storage"]
 === Storage
 
+[id="ocp-4-10-storage-alicloud-disk-csi-operator"]
+==== Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AliCloud Disk. The AliCloud Disk Driver Operator that manages this driver is generally available.
```

Comment:
@lpettyjo I believe Alibaba Cloud is going as TP in 4.10.

Reply:
Confirmed with Jan that AliCloud is GA for 4.10.

Reply:
Alibaba is GA.